### PR TITLE
add manual vol oracle

### PR DESCRIPTION
**Pull request by chuddster**

### CODE REVIEW

File: contracts/core/ManualVolOracle.sol [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract VolOracle is DSMath, AccessControl {
+    using SafeMath for uint256;
+
+    /// @dev The identifier of the role which maintains other roles.
+    bytes32 public constant ADMIN_ROLE = keccak256("ADMIN");
+
+    /**
```
Comment (chuddster) [inline contracts/core/ManualVolOracle.sol:23]: should we just gut everything else

File: contracts/core/ManualVolOracle.sol [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+     * @notice Returns the annualized standard deviation of the base currency in 10**8 i.e. 1*10**8 = 100%
+     * @return annualStdev is the annualized standard deviation of the asset
+     */
+    function annualizedVol(address pool)
+        public
+        view
+        returns (uint256 annualStdev)
+    {
+        return annualizedVols[pool];
```
Comment (chuddster) [inline contracts/core/ManualVolOracle.sol:157]: do we want to pass in already annualized or should we annualize it in here

File: contracts/core/ManualVolOracle.sol (hunk 1 above)
Comment (kenchangh) [reply contracts/core/ManualVolOracle.sol:23]: Yes, sounds good, only annualizedVol and its setter is needed

Comment (kenchangh) [reply contracts/core/ManualVolOracle.sol:23]: Let's just leave `vol` in but return a 0

File: contracts/core/ManualVolOracle.sol (hunk 2 above)
Comment (kenchangh) [reply contracts/core/ManualVolOracle.sol:157]: Let's set the annualized vol number directly

**Review by kenchangh — APPROVED** [review]
LGTM

**Comment by kenchangh** [comment]
testing 123

**Comment by kenchangh** [comment]
testing 123

**Comment by kenchangh** [comment]
Testing 123

**Comment by kenchangh** [comment]
testing

**Pull request merged**